content/ros/ROS_Nodes_Topics_Messages/home_work/README.md

```diff
@@ -0,0 +1,56 @@
+# Homework assignment
+
+In this assignment, you are asked to create a node, which for a given 
+coordinates and a given map, it tells whether the given location is occupied (there is an obstacle), free, or unknown. This is needed when a robot wants to avoid obstacles or plan a safe path to a target location.
+
+## Map file
+The map you will use for testing is shon below.
+
+![example](figures/map.png)
+
+This is a 2D occupancy grid map. It represents the environmet as a grid that consists of cells, each cell corresponds to a location in the physical space. Each cell can have a value of **100** representing occupied space, **0** representing free space, or **-1** representing unknown space. It looks like a floor plan. Before going any further, read [here](occupancy_grid_maps.md) about occupancy grid map.
```

Comment:
...consists of cells, **where** each cell corresponds to a location in ~~the~~ physical space.

...read here about occupancy grid map**s**.

Reply:
done